site/public/css/admin-gradeable.css

```diff
@@ -122,3 +122,24 @@ div.btn-container {
 #team-settings-container {
     margin-top: 10px
 }
+
+.disabledBtn{
+    background-color:#d4d4d4 ;
+    color: #5a5a5a ;
+    cursor: default;
+}
+.normalBtn{
+    background-color: #d0d0d0 ;
+    color: #111111 ;
+    border:solid 1px rgba(0,0,0,0.75);
+}
+.normalBtn:hover{
+    border: none;
+}
+
+.activeBtn{
+    background-color: #008000;
+    color: #ffffff;
+    border: none;
+}
```

Comment:
rename these to be `disabled-btn`, `normal-btn`, and `active-btn` please to match the naming scheme of the other CSS classes.

Reply:
Thanks for helping @MasterOdin .
I have changed all changes that you requested

Reply:
@pbailie  can you please review my code again